lib/segment/src/common/mod.rs

```diff
@@ -45,7 +45,14 @@ fn check_query_vector(
     vector_config: &VectorDataConfig,
 ) -> OperationResult<()> {
     match query_vector {
-        QueryVector::Nearest(vector) => check_vector_against_config(vector, vector_config)?,
+        QueryVector::Nearest(vector) => {
+            match vector {
+                Vector::Dense(dense_vector) => {
+                    check_vector_against_config(dense_vector, vector_config)?;
+                }
+                Vector::Sparse(_sparse_vector) => (), // TODO: check sparse vector config
```

Comment:
Just a wish. Can you please mark all sparse-vectors-related todos as `TODO(sparse)`. We can find by search all missed things